src/stk/io/io.cpp

```diff
@@ -138,7 +138,13 @@ namespace stk
     Volume read_volume(const std::string& filename)
     {
 #ifdef STK_ITK_BRIDGE
-        return stk::read_itk_image(filename);
+        try {
+            return stk::read_itk_image(filename);
+        }
+        catch (...) {
+            LOG(Error) << "Failed to read file " << filename;
+            return Volume();
```

Comment:
I've tried to avoid exceptions for STK just to keep it simple. Could be an idea to start allowing exceptions for the API but I think we could hold off on that for a bit.

Reply:
Sure, that makes sense. So far I have been using exceptions only internally, to keep things clear.